libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Views/SFSDKWindowManager.m

```diff
@@ -48,85 +48,26 @@
  Stash(nullify) the rootviewcontroller when the window is presented and unstash(restore) when dismissed. Extended UIWindow (SFSDKUIWindow) to handle the stash and unstash.
  Windows are created lazily and the references are removed when the windows are dismissed.
  */
-@interface SFSDKUIWindow ()
```

Comment:
This is just getting moved within the file so that it can access internal SFSDKWindowManager methods